gallery/psget/module/PreReleaseModule.md

```diff
@@ -0,0 +1,200 @@
+
+```
+
+Uninstall-Module will remove the latest version of a module when -RequiredVersion is not supplied. 
+If -RequiredVersion is specified, and is a prerelease, -AllowPrerelease must be added to the command. 
+
+``` powershell
+C:\windows\system32> Get-InstalledModule TestPackage -AllVersions
+
+Version         Name                                Repository           Description
+-------         ----                                ----------           -----------
+2.0.0-alpha1    TestPackage                         PSGallery            Package used to validate changes to the PowerShe...
+1.9.0-beta      TestPackage                         PSGallery            Package used to validate changes to the PowerShe...
+1.8.0           TestPackage                         PSGallery            Package used to validate changes to the PowerShe...
+1.1.3.2         TestPackage                         PSGallery            Package used to validate changes to the PowerShe...
+
+C:\windows\system32> Uninstall-Module TestPackage -RequiredVersion 1.9.0-beta
+Uninstall-Module : The '-AllowPrerelease' parameter must be specified when using the Prerelease string in
+MinimumVersion, MaximumVersion, or RequiredVersion.
+At line:1 char:1
++ Unnstall-Module TestPackage -RequiredVersion 1.9.0-beta
++ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    + CategoryInfo          : InvalidArgument: (:) [Uninstall-Module], ArgumentException
+    + FullyQualifiedErrorId : AllowPrereleaseRequiredToUsePrereleaseStringInVersion,Uninnstall-Module
+
+
+
+C:\windows\system32> Uninstall-Module TestPackage -RequiredVersion 1.9.0-beta -AllowPrerelease
+C:\windows\system32> Get-InstalledModule TestPackage -AllVersions
+
+Version         Name                                Repository           Description
+-------         ----                                ----------           -----------
+2.0.0-alpha1    TestPackage                         PSGallery            Package used to validate changes to the PowerShe...
+1.8.0           TestPackage                         PSGallery            Package used to validate changes to the PowerShe...
+1.1.3.2         TestPackage                         PSGallery            Package used to validate changes to the PowerShe...
+
+C:\windows\system32> Uninstall-Module TestPackage
+C:\windows\system32> Get-InstalledModule TestPackage -AllVersions
+
+Version         Name                                Repository           Description
+-------         ----                                ----------           -----------
+1.8.0           TestPackage                         PSGallery            Package used to validate changes to the PowerShe...
+1.1.3.2         TestPackage                         PSGallery            Package used to validate changes to the PowerShe...
+
+
+```
+
+
+
+## More details
+### [Prerelease Script Versions](../script/PrereleaseScript)
```

Comment:
This link does not appear to be valid.

Reply:
@sdwheeler these are not cmdlet reference articles, those will have to be separate. If you notice, NONE of the items in the powershellget area follow that structure. I can craft the reference topics, but it would be helpful if you can provide a pointer to how to do them as you expect.

Reply:
Addressed link issue in update, in both prereleasemodule and prereleasescript files